pos_default_payment_method/static/src/js/pos_default_payment_method.js

```diff
@@ -0,0 +1,25 @@
+/******************************************************************************
+ * Copyright (C) 2017-TODAY Camptocamp SA (<http://www.camptocamp.com>).
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ ******************************************************************************/
+
+odoo.define('pos_default_payment_method.activate_payment', function (require) {
+    "use strict";
+
+  var pos_models = require('point_of_sale.models');
+  var pos_screens = require('point_of_sale.screens');
+
+  // add `default_payment_method_id` to loaded pos config's fields
+  pos_models.load_fields("pos.config", "default_payment_method_id");
+
+  pos_screens.PaymentScreenWidget.include({
+    show: function(){
+      this._super();
+      // activate default payment method if any
+      if (this.pos.config.default_payment_method_id) {
+        this.click_paymentmethods(this.pos.config.default_payment_method_id[0]);
+      }
```

Comment:
Hi @simahawk. I don't understand the use case. The payment screen is displayed when you click on a journal. so why do you want to add a default payment. If I understand correctly, when user click on "Check" it will add check line and liquidity line (if liquidity is the the default payment method).

Reply:
the use case is pretty simple:
* you have several payment methods
* most of the time, depending on the POS station, you pay with the same method
* w/out this module you have to click on the payment method to enable it

It's just a way to speed up payment process when you already know how you want to pay but still allow to change the method on demand.

This is what you get w/out the module when you land on the payment page:
![screenshot from 2017-09-08 13-08-02](https://user-images.githubusercontent.com/347149/30209158-eda8cfc6-9496-11e7-84af-b49e02fcb8ce.png)
with the module instead is already activated:
![screenshot from 2017-09-08 13-08-56](https://user-images.githubusercontent.com/347149/30209171-028e31d8-9497-11e7-9b91-2546ca4c94e5.png)

Reply:
Ow thanks ! I didn't seen that was for 10.0 version.